server.go

```diff
@@ -103,6 +110,9 @@ func (srv *Server) Handler(channel string) http.HandlerFunc {
 			case <-notifier.CloseNotify():
 				srv.unregister <- sub
 				return
+			case <-maxConnTimeCh: // if MaxConnTime was not set, this is a nil channel so "<-"" is a no-op
```

Comment:
no-op seems _slightly_ inaccurate. Receiving from a `nil` channel blocks forever (which ends up being a no-op when inside of a `select`).

Reply:
I'm not sure I get your point— it _is_ inside of a select.

Reply:
Yeah. Ignore this. The `select` keyword didn't show up in the diff, but I should've inferred that it was inside a `case`.

Reply:
If you just mean the comment is unclear, I can reword it.